frontend/static/js/form.js

```diff
@@ -235,6 +247,7 @@ PreloadController.prototype = {
         break;
       case 'preloaded':
         view.setTheme('theme-green');
+        view.showIssues(issues);
```

Comment:
Still needs `if (status.bulk)` here, but the rest looks good!

Reply:
@lgarron Thanks for your patience and supervision!